src/test/java/FixedTest.java

```diff
@@ -0,0 +1,28 @@
+import duke.tasks.Fixed;
+import duke.tasks.Task;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+class FixedTest {
+
+    @org.junit.jupiter.api.Test
+    void testToString() {
+        Task t = new Fixed("read sales report", "2 hours");
```

Comment:
1) Don't use single character variable names - something like Task task would be better.
2) Follow unit test naming conventions. 

![image](https://user-images.githubusercontent.com/40201586/65157174-e7ac7500-da62-11e9-81b4-3208c59682f4.png)

Reply:
Alright, will stick to the naming conventions!